nvflare/app_opt/tracking/wandb/wandb_receiver.py

```diff
@@ -82,6 +82,13 @@ def initialize(self, fl_ctx: FLContext):
         run_name = self.kwargs["name"]
         job_id_tag = self.get_job_id_tag(run_group_id)
         wand_config = self.kwargs.get("config", {})
+
+        try:
+            wandb.login(timeout=1)
+        except Exception as e:
+            self.log_error(self.fl_ctx, f"Unsuccessful login: {e}. Will not log to wandb.")
+            return
+
```

Comment:
Have you tested this on both "online" and "offline" mode?

I wonder how do we pass server URL or something to this WandBReceiver?

Reply:
Changed to use offline mode as a default rather than not logging at all.

Reply:
thanks, for this question "how do we pass server URL or something to this WandBReceiver?" we can leave it to future PR.